exercise_2/Graph.h

```diff
@@ -0,0 +1,16 @@
+#pragma once	
+#include <iostream>
+
+class Graph
+{
+private:
```

Comment:
Usually (in C++) we start with "public", then "protected" (if not empty), then "private"